bee-network/src/tests/common/shutdown.rs

```diff
@@ -0,0 +1,10 @@
+// Copyright 2020-2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use tokio::time::{self, Duration};
+
+use std::future::Future;
+
+pub fn shutdown(secs: u64) -> Box<dyn Future<Output = ()> + Send + Unpin> {
```

Comment:
Is returning `impl Future` not a better idea? That way, the caller gets the opportunity to choose whether it wants to use an allocation to box it or not.

Reply:
you're right. Changed it.